internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go

```diff
@@ -207,6 +207,11 @@ func (k *KafkaManager) calculateAvailableAndMaxCapacityForDynamicScaling(streami
 	// of Kafkas have already been created. In this case we limit the metric to 0 anyways.
 	available := math.Max(0, maxAvailable-float64(streamingUnitsByRegion.Count))
 
+	if streamingUnitsByRegion.ClusterType == api.EnterpriseDataPlaneClusterType.String() {
+		maxAvailable = float64(streamingUnitsByRegion.MaxUnits)
+		available = maxAvailable - float64(streamingUnitsByRegion.Count)
+	}
```

Comment:
Same comment as https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1483/files#r1067884760

Reply:
@pawelpaszki we also do not need to execute https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/23d0d0031fd3e60848740dbd5cafad45afff232a/internal/kafka/internal/workers/kafka_mgrs/kafkas_mgr.go#L196 if the cluster is enterprise as the region configuration shouldn't be taken onto account for these type of clusters